src/routes/Issues/components/Issues.tsx

```diff
@@ -109,6 +110,7 @@ export default class Issues extends React.PureComponent<
 
     return (
       <div className="row issues-container">
+        <FixMeMetas title="Fixme | Issues" description="Issues page meta description" />
```

Comment:
The description isn't really descriptive, maybe something like "Open issues waiting for your contribution can be found here" (or something similar) would be more descriptive